bolt-cli/src/contracts/mod.rs

```diff
@@ -145,4 +156,12 @@ const HOLESKY_DEPLOYMENTS: Contracts = Contracts {
             m_eth: address!("accc5A86732BE85b5012e8614AF237801636F8e5"),
         },
     },
+    collateral: Token {
+        wst_eth: address!("8d09a4502Cc8Cf1547aD300E066060D043f6982D"),
+        r_eth: address!("7322c24752f79c05FFD1E2a6FCB97020C1C264F1"),
+        st_eth: address!("3F1c547b21f65e10480dE3ad8E19fAAC46C95034"),
+        w_eth: address!("94373a4919B3240D86eA41593D5eBa789FEF3848"),
+        cb_eth: address!("8720095Fa5739Ab051799211B146a2EEE4Dd8B37"),
+        m_eth: address!("e3C063B1BEe9de02eb28352b55D49D85514C67FF"),
+    },
```

Comment:
I think that turning this into an array of tuples would allow for iteration, allowing us to avoid duplicating the function call for each new token.

something like:

```rust
collateral: [
	("wst_eth".to_string(), address!("8d09a4502Cc8Cf1547aD300E066060D043f6982D"),
         ...
]

// then:
for (name, address) in collateral {
	let stake = bolt_manager.getOperatorStake(operator, address).call().await?._0;
        // ...
}

```

Reply:
Is enums the best way of working around this?

```rust
#[derive(Clone, Serialize, Deserialize, Debug)]
pub struct Token(pub Vec<(String, Address)>);

// allocations are not allowed in constants
collateral: Token(vec![
    ("wst_eth".to_string(), address!("8d09a4502Cc8Cf1547aD300E066060D043f6982D")),
```

```rust
#[derive(Clone, Serialize, Deserialize, Debug)]
// lifetime may not live long enough
// requires that `'de` must outlive `'static`
pub struct Token(pub [(&'static str, Address); 6]);

collateral: Token([
    ("wst_eth", address!("8d09a4502Cc8Cf1547aD300E066060D043f6982D")),
```

Reply:
No, it's most likely Lazy statics; constant allocs in rust are very limited unfortunately

Reply:
```rust
// cargo.toml
lazy_static = "1.4.0"
```

```rust
lazy_static! {
	public static ref HOLESKY_DEPLOYMENTS: Contracts = Contracts {
		// here you can allocate more freely
    }
}
```